hassio-google-drive-backup/backup/static/js/about.js

```diff
@@ -1,78 +1,113 @@
-var contributorsLoaded = false;
+let contributorsLoaded = false;
+
+const CONTRIBUTORS_API_URL = "https://api.github.com/repos/sabeechen/hassio-google-drive-backup/contributors";
+const CONTRIBUTOR_COMMIT_URL = "https://github.com/sabeechen/hassio-google-drive-backup/commits?author={AUTHOR}";
+const CONTRIBUTORS_COUNT_TO_SHOW = 14;
+
+const EXTRA_CONTRIBUTORS = {
+  "sabeechen": "Original author. Did most of this.",
+  "ericmatte": "Redesigned the UI with divine HTML and CSS wizardry.",
+  "jhampson-dbre": "Fixed a really nasty timing bug.",
+};
+
+function createElementFromHTML(htmlString) {
+  var div = document.createElement("div");
+  div.innerHTML = htmlString.trim();
+  return div.firstChild;
+}
+
+function showElement(id) {
+  document.getElementById(id).classList.remove("default-hidden");
+}
+
+function hideElement(id) {
+  document.getElementById(id).classList.add("default-hidden");
+}
+
+async function preloadImage(url) {
```

Comment:
This function is what allow us to wait for all image to be loaded before removing the loading bar.

Reply:
Oh, thats nice.  I like it.  I'll use this trick more in the future.